### ISSUE TITLE
Robots Metas (Index, noindex, follow, nofollow..)

### ISSUE THREAD

**Issue by toanna** [artesaos/seotools#106]
Should we add some functions for Robots Metas?

**Comment by klimov-paul** [comment]
I don't get it: what is wrong with `SEOMeta::setRobots()` for you?

You can easily setup any combination of robots directive based on your application logic.
For example:

```php
class MyController extends Controller
{
    public function publicPage()
    {
        SEOMeta::setRobots('index,follow')
    }

    public function privatePage()
    {
        SEOMeta::setRobots('noindex,nofollow')
    }
}
```

The practical example is setting it at config file, for example:

```php
<?php
// file 'config/seotools.php'
return [
'meta' => [
        'defaults'       => [
            // ...
            'robots'       => env('ALLOW_ROBOTS', env('APP_ENV') === 'production') ? 'index,follow' : 'noindex,nofollow',
        ],
        //...
];
```

**Comment by klimov-paul** [comment]
I consider this issue to be resolved by #132
